builtin/providers/datadog/resource_datadog_monitor.go

```diff
@@ -393,16 +393,9 @@ func resourceDatadogMonitorDelete(d *schema.ResourceData, meta interface{}) erro
 	return nil
 }
 
-func resourceDatadogImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Could you give some context as to why you removed this? `ImportStatePassthrough` just does a return of `[]*schema.ResourceData{d}` and `nil`. What happens when the monitor was manually removed a user?